addons/mongodb/templates/componentdefinition.yaml

```diff
@@ -195,3 +193,28 @@ spec:
           name: root
           optional: false
           password: Required
+    - name: KB_SERVICE_PORT  
+      value: "27017"
```

Comment:
remove the value?

Reply:
it is the default port for mongodb, if hostnetwork is not enabled, the env will use this value.

Reply:
got it